zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -520,7 +520,7 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, environment, region,
             'id': 'pg-{}[{}]'.format(service.name, cluster_id),
             'type': POSTGRESQL_CLUSTER_TYPE,
             'kube_cluster': cluster_id,
-            'alias': alias,
+            'account_alias': alias,
```

Comment:
Why changing to``account_alias``?

Reply:
because we call it account_alias in other entities too? And it is def. not an alias for a postgresql_cluster_member

Reply:
it is actually called alias: https://github.com/zalando-zmon/zmon-agent-core/pull/38/files#diff-654e8467ef59c6375e05b4554f0c89fcR464
and I think lots of Kubernetes checks depend on that.

Reply:
Local entity uses account_alias and that sounds like a pretty good name for it.